docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -94,41 +94,86 @@ Any dependencies specified in either `requirements.txt` or `setup.py` will be in
 
 ---
 
-## Customizing the container
+## Customizing the runtime environment
 
-Many apps will work fine with the default Dagster Cloud Serverless setup. However, some apps may need to make changes to the base image, either to use a different OS, different Python version, or install some native dependencies. You can customize your image through our Serverless lifecycle hooks, and/or changing the base image.
+Dagster Cloud Serverless packages your code as [PEX](https://pex.readthedocs.io) files and deploys them on Docker images. Many apps will work fine with the default Dagster Cloud Serverless setup. However, some apps may need to make changes to the runtime environment, either to include data files, use a different OS, different Python version, or install some native dependencies. You can customize the runtime environment using various methods described below.
 
-### Lifecycle hooks
+### Including data files
 
-This method is the easiest to set up, and does not require setting up any additional infrastructure.
+To add data files to your deployment, use the [Data Files Support](https://setuptools.pypa.io/en/latest/userguide/datafiles.html) built into Python's setup.py. This requires adding a `package_data` or `include_package_data` keyword in the call to `setup()` in `setup.py`.
 
-In the root of your repo, you can provide two optional shell scripts: `dagster_cloud_pre_install.sh` and `dagster_cloud_post_install.sh`. These will run before and after Python dependencies are installed. They are useful for installing any non-Python dependencies or otherwise configuring your environment.
+### Using a different Python version
 
-### Changing the base image
+The default version of Python for Serverless deployments is Python 3.8. Versions 3.7, 3.9 and 3.10 are also supported. You can specify the version you want by updating your GitHub workflow or using the `--python-version` command line argument:
```

Comment:
any particular reason the default isn't the latest version?

Reply:
No reason. It's just the one we use. Should I change it to 3.10?

Reply:
Probably the most compelling reason is that all of our internal tests run using 3.8.